lib/user.rb

```diff
@@ -0,0 +1,24 @@
+class User
+
+attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+    @jokes
+  end
```

Comment:
good job here; by default, shoveling an item onto a list will return that new list, so line 12 isn't necessary here